### DISCUSSION TITLE
Advise for the betterment of Core coin

### DISCUSSION THREAD

**Discussion by Oluwoleoluma** [core-coin/web-blog#40] [Ideas]
I will advise Core should gain strong ground and should be use as a mode of payment just like Bitcoin, so it can increase in value and always waxing strong like Bitcoin

**Comment by Oluwoleoluma** [comment]
This a great idea

**Comment by Oluwoleoluma** [comment]
Please let the developers of Core look into these advise

**Comment by rastislavcore** [comment]
> I will advise Core should gain strong ground and should be use as a mode of payment just like Bitcoin, so it can increase in value and always waxing strong like Bitcoin

We are working on that. Multiple tasks are done and few are in front of us.

**Comment by Oluwoleoluma** [comment]
Please Will you advise me,or can you suggest if it is advisable to invest
more of my millions in Core? And please I will love to ask or know what
will be the price of Core by November 2024

Thank you
Kehinde Oluwole.

**Reply by rastislavcore** [reply]
Thank you for your interest in Core. Please, feel free discuss this topic in coretalk.space or Discord/Telegram channels. Here we are focusing mainly about the blog discussion. But I will be happy to assist you best I can.

**Comment by Oluwoleoluma** [comment]
Awesome, thank you so much!

And please where or how can I get Core telegram channel
